src/vector/jitsi/index.ts

```diff
@@ -46,6 +46,7 @@ let jitsiAuth: string;
 let roomId: string;
 let openIdToken: IOpenIDCredentials;
 let roomName: string;
+let startAudioOnly: string;
```

Comment:
boolean? Worried that `"false"` gets treated as true.

Reply:
🤦 thanks